scripts/rsync.Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:20.04
+
+ARG DOMINO_UID=12574
+ARG DOMINO_USER=domino
+ARG DOMINO_GID=12574
+ARG DOMINO_GROUP=domino
+
+ARG RSYNC_SSH_PORT=2223
+
+ARG DOMINO_DIR=/opt/domino
+ARG DOMINO_SSH_DIR=${DOMINO_DIR}/etc/ssh
+
+ARG SSHD_CONFIG=${DOMINO_SSH_DIR}/sshd_config
+ARG AUTHORIZED_KEYS_PATH=/etc/mpi/authorized_keys
+
+RUN \
+	apt-get update && \
+	apt-get -y install \
```

Comment:
Make sure to clear the `apt` cache to make the layer as small as possible. 

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Reply:
Oh, right. And possibly some other cleanup.